### PR TITLE
(chore) align React and Vue docs; align prop/param descriptions

### PR DESCRIPTION
**Pull request by victoriaxyz**
- fix code examples (e.g., missing chaining ops), filenames, code comments
- add `Returns` and `Params headings`, and more descriptive example headings
- replace all tables with `<Properties />` table
- align content, rewrite descriptions and intro paragraphs
- remove `More resources` section

### CODE REVIEW

File: docs/references/react/use-sign-up.mdx [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
 title: '`useSignUp()`'
-description: Clerk's useSignUp() hook gives you access to the SignUp object, which allows you to check the current state of a sign-up.
+description: Access and manage the current user's sign-up state in your React application with Clerk's useSignUp() hook.
 ---
 
-The `useSignUp()` hook gives you access to the [`SignUp`](/docs/references/javascript/sign-up/sign-up) object, which allows you to check the current state of a sign-up. This is also useful for creating a [custom sign-up flow](#create-a-custom-sign-up-flow-with-use-sign-up).
+The `useSignUp()` composable provides access to the [`SignUp`](/docs/references/javascript/sign-up/sign-up) object, which allows you to check the current state of a sign-up attempt and manage the sign-up flow. You can use this to create a [custom sign-up flow](/docs/custom-flows/overview#sign-up-flow).
```
Comment (alexisintech) [inline docs/references/react/use-sign-up.mdx:6]:
```suggestion
The `useSignUp()` hook provides access to the [`SignUp`](/docs/references/javascript/sign-up/sign-up) object, which allows you to check the current state of a sign-up attempt and manage the sign-up flow. You can use this to create a [custom sign-up flow](/docs/custom-flows/overview#sign-up-flow).
```

File: docs/references/react/use-sign-in.mdx [hunk 2]
```diff
@@ -53,35 +53,60 @@ The `useSignIn()` hook provides access to the [`SignIn`](/docs/references/javasc
 
 ## How to use the `useSignIn()` hook
 
-### Check the current state of a sign-in with `useSignIn()`
+### Check the current state of a sign-in
 
-Use the `useSignIn()` hook to check the current state of a sign-in.
+Use `useSignIn()` to check the current sign-in status and display the value in the UI.
```
[the remark this answers is not in the document]
Comment (alexisintech) [reply docs/references/react/use-sign-in.mdx:58]: ahh actually, let's align it with the usesignup suggestion you added, to keep them consistent

```suggestion
Use `useSignIn()` to access the `SignIn` object and check the current state of a sign-up.
```

File: docs/references/react/use-sign-in.mdx [hunk 3]
```diff
@@ -53,35 +53,60 @@ The `useSignIn()` hook provides access to the [`SignIn`](/docs/references/javasc
...
+  - `string`
+
+  One of the following [second factor verification strategies](/docs/references/javascript/sign-in/second-factor) is missing: `phone_code` or `totp`. This value is optional.
+
+  ---
+
+  - `needs_identifier`
+  - `string`
 
-### Create a custom sign-in flow with `useSignIn()`
```
Comment (alexisintech) [inline docs/references/react/use-sign-in.mdx:85]: these sections need to stay - they were added a result of user feedback.
we can rename to "Example" if that makes more sense

File: docs/references/react/use-sign-up.mdx [hunk 4]
```diff
@@ -53,33 +53,46 @@ The `useSignUp()` hook gives you access to the [`SignUp`](/docs/references/javas
...
+  The user has been created and the custom flow can proceed to `setActive()` to create a session.
+
+  ---
+
+  - `abandoned`
+  - `string`
+
+  The sign-up attempt will be abandoned if it was started more than 24 hours previously.
 
-### Create a custom sign-up flow with `useSignUp()`
```
Comment (alexisintech) [inline docs/references/react/use-sign-up.mdx:83]: these sections need to stay - they were added a result of user feedback.
we can rename to "Example" if that makes more sense

File: docs/references/react/use-user.mdx [hunk 5]
```diff
@@ -1,49 +1,47 @@
...
+  - `isSignedIn`
   - `boolean`
 
-  A boolean that until Clerk loads and initializes, will be set to `false`. Once Clerk loads, `isLoaded` will be set to `true`.
+  A boolean that returns `true` if the user is signed in.
 
   ---
 
   - `user`
   - [`User`](/docs/references/javascript/user/user)
```
Comment (alexisintech) [inline docs/references/react/use-user.mdx:26]: that backslash might not be needed - i'd check locally, see how its rendered with and without

```suggestion
  - <code>[User](/docs/references/javascript/user/user) \| null</code>
```

File: docs/references/react/use-user.mdx [hunk 6]
```diff
@@ -1,49 +1,47 @@
...
 
 ## How to use the `useUser()` hook
 
-### Retrieve the current user data with the `useUser()` hook
-
-The following example demonstrates how to use the `useUser()` hook to access the `user` object, which includes the current user's data, like the user's full name. The `isLoaded` and `isSignedIn` properties are used to handle the loading state and to check if the user is signed in, respectively.
+### Get the current user
 
-For more information on the `User` object, see the [`reference`](/docs/references/javascript/user/user).
+Use `useUser()` to access the [`User`](/docs/references/javascript/user/user) object, which contains the current user's data such as their full name. The `isLoaded` and `isSignedIn` values are used to handle the loading state and to check if the user is signed in, respectively.
```
Comment (alexisintech) [inline docs/references/react/use-user.mdx:35]: actually, I don't see anything wrong with the copy before, and its a common syntax structure we use across the docs: "the following example demonstrates..."
if you feel this is cleaner, that's fine, but isLoaded and isSignedIn aren't values, they're properties that are set to boolean values.

File: docs/references/react/use-user.mdx [hunk 7]
```diff
@@ -55,20 +53,18 @@ export default function Home() {
 }
 ```
 
-### Update the current user data with the `useUser()` hook
+### Update user data
 
-The following example demonstrates how to use the `useUser()` hook to access the `user` object, which includes the `update` method for updating the user's data.
+Use [`update()`](/docs/references/javascript/user/user#update) to update the current user's information.
```
Comment (alexisintech) [inline docs/references/react/use-user.mdx:58]: this doesn't provide all the context that it should.

File: docs/references/react/use-user.mdx [hunk 8]
```diff
@@ -83,28 +79,26 @@ export default function Home() {
 
   return (
     <>
-      <button onClick={updateUser}>Click me to update your name</button>
+      <button onClick={updateUser}>Update your name</button>
```
Comment (alexisintech) [inline docs/references/react/use-user.mdx:82]: nice update!

File: docs/references/react/use-user.mdx [hunk 9]
```diff
@@ -83,28 +79,26 @@ export default function Home() {
...
 }
 ```
 
-### Reload user data with the `useUser()` hook
-
-To retrieve the latest user data after updating the user elsewhere, use the `user.reload()` method.
+### Reload user data
 
-For more information on the `reload()` method, see the [`User`](/docs/references/javascript/user/user#reload) reference.
+Use [`reload()`](/docs/references/javascript/user/user#reload) to get the latest user's information via an external API endpoint.
```
Comment (alexisintech) [inline docs/references/react/use-user.mdx:92]: it should provide more context.
the useUser() hook gives access to the user object, which is how you access the reload method.
useUser() does not give you direct access to the reload method

File: docs/quickstarts/react.mdx [hunk 10]
```diff
@@ -199,22 +199,3 @@ React Router can be integrated with Clerk in two ways:
 
 - As a framework: Use Clerk's built-in [React Router integration](/docs/quickstarts/react-router)
 - As a library: Manually integrate React Router into your Clerk application using [library mode](/docs/references/react-router/library-mode)
-
-## More resources
```
[the remark this answers is not in the document]
Comment (victoriaxyz) [reply docs/quickstarts/react.mdx:203]: My info might be outdated on this, but I thought we were removing these sections (or maybe that was only the case for specific docs like the OAuth ones).

File: docs/references/react/use-clerk.mdx [hunk 11]
```diff
@@ -1,20 +1,20 @@
...
 
-## `useClerk()` returns
+## Returns
 
 The `useClerk()` hook returns the `Clerk` object, which includes all the methods and properties listed in the [`Clerk` reference](/docs/references/javascript/clerk/clerk).
 
 ## How to use the `useClerk()` hook
 
-```tsx {{ filename: 'home.tsx' }}
+```tsx {{ filename: 'src/Home.tsx' }}
```
[the remark this answers is not in the document]
Comment (victoriaxyz) [reply docs/references/react/use-clerk.mdx:17]: Yes, will do!

File: docs/quickstarts/react.mdx (hunk 10 above)
[the remark this answers is not in the document]
Comment (alexisintech) [reply docs/quickstarts/react.mdx:203]: they were asked to be added to the quickstarts - hopefully we didn't accidentally remove these from all the quickstarts 😥

[the remark this answers is not in the document]
Comment (victoriaxyz) [reply docs/quickstarts/react.mdx:203]: No worries, I can add them back!

[the remark this answers is not in the document]
Comment (victoriaxyz) [reply docs/quickstarts/react.mdx]: https://github.com/clerk/clerk-docs/pull/1814/commits/872d48efd82e9deecc4f3b53a13de9661787fdbd

File: docs/references/react/use-auth.mdx [hunk 12]
```diff
@@ -1,111 +1,114 @@
...
   }
 
-  const fetchDataFromExternalResource = async () => {
-    const token = await getToken()
-    // Add logic to fetch your data
-    return data
-  }
-
-  return <div>...</div>
+  return <div>{/* Add your component content here */}</div>
```
[the remark this answers is not in the document]
Comment (victoriaxyz) [reply docs/references/react/use-auth.mdx]: https://github.com/clerk/clerk-docs/pull/1814/commits/826b80d6d5ffa8796755dc663b34e8c45a7ad461

File: docs/references/react/use-sign-in.mdx (hunk 3 above)
Comment (victoriaxyz) [reply docs/references/react/use-sign-in.mdx:85]: https://github.com/clerk/clerk-docs/pull/1814/commits/70af8fedfe4a526a54f434837cf77c6e25921384

File: docs/references/react/use-clerk.mdx (hunk 11 above)
[the remark this answers is not in the document]
Comment (victoriaxyz) [reply docs/references/react/use-clerk.mdx:19]: https://github.com/clerk/clerk-docs/pull/1814/commits/68a0e44899320a4c33331a7465db40f1fa4ff7af

**Comment by victoriaxyz** [comment]
@alexisintech 

- [put back More resources section](https://github.com/clerk/clerk-docs/pull/1814/commits/872d48efd82e9deecc4f3b53a13de9661787fdbd)
- [align definitions across all files](https://github.com/clerk/clerk-docs/pull/1814/commits/b0a6c9acc1dc0ccd1f0da78df03bb9f6642484cb)
- [fix useUser() example](https://github.com/clerk/clerk-docs/pull/1814/commits/b84088adca025883895f5c86c4d21b3d11c3b3f6)
- [update example intros](https://github.com/clerk/clerk-docs/pull/1814/commits/be8c5719fd978ebc53665c29e9456adeec2958e3)
- [add example button to fetch data](https://github.com/clerk/clerk-docs/pull/1814/commits/826b80d6d5ffa8796755dc663b34e8c45a7ad461)
- [put back sections for custom sign-in and sign-up flows](https://github.com/clerk/clerk-docs/pull/1814/commits/70af8fedfe4a526a54f434837cf77c6e25921384)
- [fix code filenames to match quickstart](https://github.com/clerk/clerk-docs/pull/1814/commits/68a0e44899320a4c33331a7465db40f1fa4ff7af)

File: docs/references/react/use-auth.mdx [hunk 13]
```diff
@@ -1,111 +1,118 @@
...
+> For Next.js applications, it's recommended to use the [`auth()`](/docs/references/nextjs/auth) helper instead of `useAuth()`. Since `auth()` must be used in Server Components, you'll need to pass auth data to Client Components as needed. If you prefer `useAuth()`, you must pass the `dynamic` prop to `<ClerkProvider>`, but be aware this switches the app to dynamic rendering. Learn more [here](/docs/references/nextjs/rendering-modes).
 
-The following example demonstrates how to use the `useAuth()` hook to access the current auth state, like whether the user is signed in or not. It also demonstrates a basic example of how you could use the `getToken()` method to retrieve a session token for fetching data from an external resource.
+The following example uses the `useAuth()` hook to fetch data from an external API using a session token. The `getToken()` method is used to retrieve the token.
```
Comment (alexisintech) [inline docs/references/react/use-auth.mdx:83]: I still wouldn't say this is correct, as per my last comment:
You don't use useAuth() to fetch data using a session token. You use useAuth() to access the getToken() method, which gives you your session token. Then you can use your session token to fetch data

File: docs/references/react/use-clerk.mdx [hunk 14]
```diff
@@ -1,20 +1,22 @@
...
 
-## `useClerk()` returns
+## Returns
 
 The `useClerk()` hook returns the `Clerk` object, which includes all the methods and properties listed in the [`Clerk` reference](/docs/references/javascript/clerk/clerk).
 
 ## How to use the `useClerk()` hook
 
-```tsx {{ filename: 'home.tsx' }}
+The following example demonstrates how to use the `useClerk()` hook to open the sign-in modal. The `clerk` object is used to call the [`openSignIn()`](/docs/references/javascript/clerk/clerk#opensignin) method to open the sign-in modal.
```
Comment (alexisintech) [inline docs/references/react/use-clerk.mdx:17]:
```suggestion
The following example uses the `useClerk()` hook to access the `clerk` object. The `clerk` object is used to call the [`openSignIn()`](/docs/references/javascript/clerk/clerk#sign-in) method to open the sign-in modal.
```

File: docs/references/react/use-sign-up.mdx [hunk 15]
```diff
@@ -53,32 +53,49 @@ The `useSignUp()` hook gives you access to the [`SignUp`](/docs/references/javas
 
 ## How to use the `useSignUp()` hook
 
-### Check the current state of a sign-up with `useSignUp()`
+### Check the current state of a sign-up
 
-Use the `useSignUp()` hook to access the `SignUp` object and check the current state of a sign-up.
+The following example uses the `useSignUp()` hook to access the [`SignUp`](/docs/references/javascript/sign-up/sign-up) object and check the current status of a sign-up.
```
Comment (alexisintech) [inline docs/references/react/use-sign-up.mdx:58]: let's make this match the usesignin() example description

**Comment by alexisintech** [comment]
we need to merge this PR: https://github.com/clerk/clerk-docs/pull/1824 into this one

File: docs/references/react/use-sign-up.mdx (hunk 15 above)
Comment (victoriaxyz) [reply docs/references/react/use-sign-up.mdx]: https://github.com/clerk/clerk-docs/pull/1814/commits/bce575f331033786c742c87ac7abb911c5f9f128

File: docs/references/react/use-auth.mdx (hunk 13 above)
Comment (victoriaxyz) [reply docs/references/react/use-auth.mdx]: https://github.com/clerk/clerk-docs/pull/1814/commits/d241fcb7a57c0f63d82c4b846d398fc5f850a094

File: docs/references/react/use-auth.mdx [hunk 16]
```diff
@@ -1,111 +1,118 @@
...
+> For Next.js applications, it's recommended to use the [`auth()`](/docs/references/nextjs/auth) helper instead of `useAuth()`. Since `auth()` must be used in Server Components, you'll need to pass auth data to Client Components as needed. If you prefer `useAuth()`, you must pass the `dynamic` prop to `<ClerkProvider>`, but be aware this switches the app to dynamic rendering. Learn more [here](/docs/references/nextjs/rendering-modes).
 
-The following example demonstrates how to use the `useAuth()` hook to access the current auth state, like whether the user is signed in or not. It also demonstrates a basic example of how you could use the `getToken()` method to retrieve a session token for fetching data from an external resource.
+The following example uses the `useAuth()` hook to access the `getToken()` method which is used to fetch data from an external API.
```
Comment (alexisintech) [inline docs/references/react/use-auth.mdx:83]:
```suggestion
The following example demonstrates how to use the `useAuth()` hook to access the current auth state, like whether the user is signed in or not. It also includes a basic example for using the `getToken()` method to retrieve a session token for fetching data from an external resource.
```

**Review by alexisintech — APPROVED** [review]

**Pull request merged**